widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/AdditionalDetailsPanel.java

```diff
@@ -0,0 +1,98 @@
+package org.pentaho.mantle.client.dialogs.scheduling;
+
+import com.google.gwt.user.client.ui.*;
+import org.pentaho.gwt.widgets.client.controls.ErrorLabel;
```

Comment:
You have some checkstyle issues. Please run our checkstyle in Intellij using these instructions:
the README.md file in https://github.com/pentaho/pentaho-coding-standards